src/modules/DialogOverlay.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <Dialog
+    v-if="isDialogOpen"
+    id="widget-dialog"
+    :button-close-label="buttonCloseLabel"
+    @toggle-dialog="toggleDialog"
+  >
+    <template #header> <slot name="header"></slot> </template>
+    <template #body>
+      <Tabs :tabs="tabsData">
+        <template #default="{ activeTabId }">
+          <Tab tab-id="12months" :active-tab-id="activeTabId">
+            <DataList :contents="tabsContents('12months')" />
+          </Tab>
+          <Tab tab-id="18months" :active-tab-id="activeTabId">
+            <DataList :contents="tabsContents('18months')" />
+          </Tab>
+        </template>
+      </Tabs>
+      <Accordion id="widget-terms" :content="accordianData">
+        Terms & Conditions
+      </Accordion>
+      <slot name="footer"></slot>
```

Comment:
Nit: This slot can probably be self closing

Reply:
great eye!
done